modules/editor-app-bar/module.php

```diff
@@ -15,7 +15,6 @@ class Module extends BaseModule {
 	const PACKAGES = [
 		'editor-app-bar',
 		'editor-documents',
-		'editor-panels',
```

Comment:
I think documents should be moved as well, and the site-navigation should be in its own module